README.md

```diff
@@ -1,18 +1,52 @@
-# Home Assistant Operating system
+# Home Assistant Operating System
 
-Home Assistant Operating System (HassOS) is based on [buildroot](https://buildroot.org/). It's a hypervisor for Docker and supports various kind of hardware. It is also available as virtual appliance for different virtualization solutions. The whole system is optimized for hosting [Home Assistant](https://www.home-assistant.io) and its features (to be precise, the [Add-ons](https://www.home-assistant.io/addons/)). You can update the system by using OTA updates or offline updates.
+Home Assistant Operating System (HassOS) is an operating system optimized for hosting [Home Assistant](https://www.home-assistant.io) and its [Add-ons](https://www.home-assistant.io/addons/).
 
-This is an embedded Linux which works different than a normal Linux distribution. The system is designed to run with minimal I/O and is optimized for its tasks.
+HassOS is uses Linux as a hypervisor for Docker, allowing each component of Home Assistant including add-ons to run in separate docker containers. It uses embedded Linux based on [buildroot](https://buildroot.org/), which is different to a "normal" Linux, and it is optimized for running Home Assistant on IoT devices.
 
-If you don't have experience with embedded systems, buildroot or the build process Linux distributions, then please read up on those topics. All provided documentation here is focusing on developers with a background on embedded systems or a strong understanding of the internal workings of operating systems.
+## Features
 
-## Focus
+- Lightweight and memory-efficient
+- Minimized I/O
+- Over The Air (OTA) updates
+- Offline updates
+- Modular using Docker
 
-- Barebox as bootloader on EFI
-- U-Boot as bootloader
-- Linux/Buildroot LTS
-- RAUC for OTA updates
-- SquashFS LZ4 as filesystem
-- Docker-CE
-- AppArmor protected
-- ZRAM LZ4 for `/tmp`, `/var` and swap
+## Supported hardware
+
+- Raspberry Pi
+- Hardkernel ODROID
+- Intel NUC
+- Asus Tinker Board
+- Virtual appliances
+
+See full list and specific models [here]
+
+## Getting Started
+
+If you just want to use Home Assistant the official [getting started guide](https://www.home-assistant.io/getting-started/) and [installation instructions](https://www.home-assistant.io/hassio/installation/) take you through how to download HassOS and get it running on your machine.
```

Comment:
Replace HassOS with Home Assistant Operating System here, as well as in the sentence and title below.

Reply:
resolved in 57cbd3f